rasa/server.py

```diff
@@ -692,6 +699,16 @@ def create_app(
                 "TrainingError",
                 "An unexpected error occurred during training. Error: {}".format(e),
             )
+        finally:
+            with app.active_training_processes.get_lock():
+                app.active_training_processes.value -= 1
+
+    @app.get("/model/train/status")
+    @requires_auth(app, auth_token)
+    async def training_status(request: Request):
+        """Get status of ongoing training jobs."""
+
+        return response.json({"n_jobs": app.active_training_processes.value})
```

Comment:
why not make this part of the `/status` endpoint?

Reply:
The purpose of `/status` seemed to be "Information about the currently loaded Rasa model", so I thought of not polluting the semantics of that with information not specific to the currently loaded model.

Reply:
I'd also prefer adding it to the `/status` endpoint

Reply:
@wochinge So it would no longer be "status of the currently loaded model". Is that okay?

Reply:
Imho yes, we could call it simply `Status of the server`?